packages/web-react/src/components/CheckboxField/__tests__/CheckboxField.test.tsx

```diff
@@ -51,17 +51,38 @@ describe('CheckboxField', () => {
     expect(element).toHaveClass('CheckboxField__input');
   });
 
-  it('should have message', () => {
+  it('should have helper text', () => {
+    const dom = render(<CheckboxField id="checkbox" label="Label" helperText="text" />);
+
+    const element = dom.container.querySelector('.CheckboxField__helperText') as HTMLElement;
+    expect(element.textContent).toBe('text');
+  });
+
+  it('should have validation message', () => {
     const dom = render(<CheckboxField id="checkbox" label="Label" message="text" />);
 
     const element = dom.container.querySelector('.CheckboxField__message') as HTMLElement;
     expect(element.textContent).toBe('text');
   });
 
-  it('should have error classname', () => {
-    const dom = render(<CheckboxField id="checkbox" label="Label" validationState="error" />);
+  it('should have danger classname', () => {
```

Comment:
I think here should be used ValidationStatePropsTest. See the colors props test for example.

Reply:
done